apps/fabric-example/android/app/build.gradle

```diff
@@ -116,6 +116,7 @@ dependencies {
     }
 
     implementation("androidx.core:core-splashscreen:1.0.1")
+    implementation project(':@shopify-flash-list')
```

Comment:
Needed to add [manual linking](https://shopify.github.io/flash-list/docs/guides/manual-linking/) for both iOS and Android to make it work

Reply:
Probably related to the monorepo. It was unable to find the native `AutoLayoutView` without manual linking.

Reply:
There is an [issue](https://github.com/Shopify/flash-list/issues/1221) on the flash-list repo. It seems that something is broken and only manual linking works.
We decided with @piaskowyk to merge it as it is for now and remove manual linking once the flash-list issue is fixed.

Reply:
I added flash list as a dependency only to the common app and not to fabric/paper example. This made RN unable to find native dependencies.